src/conn/conn_api.c

```diff
@@ -3120,6 +3118,17 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler, const char *c
     /* Start the worker threads and run recovery. */
     WT_ERR(__wt_connection_workers(session, cfg));
 
+    /*
+     * If the user wants to verify WiredTiger metadata, verify the history store now that the
+     * metadata table may have been salvaged and eviction has been started and recovery run.
+     */
+    if (verify_meta) {
```

Comment:
If we are verifying the metadata function, my thinking is that there is no point to verifying the history store. What are your thoughts? @etienneptl

Reply:
After discussing this, the team decided on the following:

- Perform HS validation when `verify_meta` is `true`.
- Perform HS validation when `verify_hs` is `true`. This will be done in a new ticket: WT-10841.

Reply:
Sounds good to me!